app/src/main/java/org/simple/clinic/home/overdue/OverdueAppointmentRow.kt

```diff
@@ -128,6 +132,9 @@ data class OverdueAppointmentRow(
 
     holder.diagnosisLabelContainer.visibleOrGone(showDiagnosisLabel)
     holder.diagnosisTextView.text = diagnosisText(context)
+
+    holder.patientTransferredContainer.visibleOrGone(isAppointmentAtAssignedFacility.not())
```

Comment:
Can we add the name to the parameter here to make it more clear?

Reply:
Done